package.json

```diff
@@ -29,18 +29,17 @@
   "bugs": {
     "url": "https://github.com/tachyons-css/react-native-style-tachyons/issues"
   },
-  "main": "lib/index.js",
+  "main": "src/index.js",
   "dependencies": {
     "color": "^3.0.0",
     "css-color-names": "0.0.4",
     "lodash": "^4.17.5"
   },
   "files": [
-    "lib/"
+    "src/"
```

Comment:
I don't want to ship source, since that means that RNT depends on the fragile babel-builds. Just a stray `.babelrc` in some parent-directory of your react-native-app could break RNT in ways we cannot predict.